missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/PlayerListener.java

```diff
@@ -153,6 +157,34 @@ public void onLeaveWithStuff(PlayerArenaLeaveEvent event) {
         MissileWars.getInstance().getSignRepository().getSigns(game).forEach(MWSign::update);
     }
 
+    // cancel anvil interaction to block renaming, and thus changing missiles
+    @EventHandler
+    public void onClick(InventoryClickEvent event) {
+        Game game = getGame(event.getWhoClicked().getLocation());
+        if (game != null) {
+            if (event.getInventory().getType() == InventoryType.ANVIL) {
+                event.setCancelled(true);
+            }
+        }
+    }
+
+    // Internal stuff
+
+    @EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
+    public void onMaxPlayers(PrePlayerArenaJoinEvent event) {
+        if (event.getGame().getPlayers().size() >= event.getGame().getLobby().getMaxSize() && event.getGame().getState() == GameState.LOBBY)
```

Comment:
Simplify if clause and use guard clause

Reply:
I down't reworked this event in this PR. Only the "event" variable. Reworking coming with the next PR.